hydra-main/src/main/java/com/addthis/hydra/job/web/resources/JobsResource.java

```diff
@@ -779,7 +779,7 @@ public Response saveJob(@QueryParam("pairs") KVPairs kv,
 
     /**
      *
-     * @param jobId         job id
+     * @param job           job id
```

Comment:
right, I was looking at `@QueryParam("job")`, that was wrong and will be fixed in the next push.